nevergrad/benchmark/test_experiments.py

```diff
@@ -25,14 +26,11 @@
 
 @testing.parametrized(**{name: (name, maker) for name, maker in experiments.registry.items()})
 def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[experiments.Experiment]]) -> None:
-    if "image_" in name and "/circleci/" in str(Path(__file__)):
```

Comment:
with this, the code would not even be run once, which I would rather avoid because that will eventually break for sure if it does not